data/guns.json

```diff
@@ -459,5 +459,43 @@
 		"symbol": "-",
 		"type": "auto",
 		"name": "jugGun"
+	},
+	"revolver":{
+		"bullet": {
+			"damage": 110,
+			"speed": 100,
+			"step": 1,
+			"lifeTime": 1400,
+			"explosive": false,
+			"fireTick": 0
+		},
+		"barrel": {
+			"len": 25,
+			"color": {
+				"r": 0,
+				"g": 0,
+				"b": 0
+			}
+		},
+		"playerMod": {
+			"scl": 1,
+			"speed": 2,
```

Comment:
Speed can defenitely be changed, probably set to around normal or slightly faster.